docs/source/contrib_docs/doc-new-translation.md

```diff
@@ -0,0 +1,87 @@
+# Supporting document translations
+
+**Work in progress**: Rough notes to be replaced with a first draft
+(https://github.com/jupyter/jupyter/issues/511#issuecomment-644221822)
+
+- Define what we mean by translation
+- Explain that we're using a workflow borrowed from Python and Django in this jupyter/jupyter repo
+  and others (docker-stacks) to support translation
+- Explain that the same system can be setup for other jupyter projects
+
+## Overview
+
+- Diagram of the translation CI/CD flow _after_ setup
+- Sumamry: travis creates en_US po files from English documentation, transifex watches for po file
```

Comment:
a quick question here: is this something integrated with travis? Or something we can use with other CI/CD systems? I'm noticing a gradual shift in contributor knowledge from "knowing travis" to "knowing CircleCI / GitHub Actions", and I wonder if this will be more maintainable in the longer term if we use one of those

Reply:
It's a series of shell commands in the .travis.yaml, easily portable to other CI systems like GitHub Actions or Circle.